src/Strings/Strings.js

```diff
@@ -238,6 +243,11 @@ export const strings = new LocalizedStrings({
         showHideCols: 'Mostrar/Ocultar Columnas',
         copyTable: 'Copiar tabla al portapapeles',
         downloadCSV: 'Descargar como CSV',
+		downloadingCSV: 'Descargando tu archivo',
+		downloadedCSVSuccessully: 'Su archivo se ha descargado exitosamente con la consulta',
+		downloadedCSVWarning: 'ADVERTENCIA: El archivo que ha solicitado es más grande que',
+		partialCSVDataWarning: 'Esto excede el tamaño máximo de descarga y solo recibirá datos parciales.',
+		fetchingCSV: 'Obteniendo su archivo',
```

Comment:
We might want to get Oscar to review the translations

Reply:
good idea!